dev-manuals/test-plans/overworld/u-overworld-20-loading-screen.md

```diff
@@ -0,0 +1,35 @@
+# Overworld: Loading screen (`u.overworld-20`)
+
+Version: v1.0, 2022-08-27, create test plan \
+Author: Florian Wüst, Gilian Rehm
+
+## Description
+
+This use case verifies that the loading screen works as intended.  
+
+## Precondition
+
+The tester is logged in
```

Comment:
There is missing something, otherwise the tester can already be in the overworld where no loading screen is.